examples/imdb/imdb/requirement.yaml

```diff
@@ -0,0 +1,115 @@
+absl-py==0.12.0
+astunparse==1.6.3
+attrs==21.4.0
+cached-property==1.5.2
```

Comment:
Are you sure its a yaml file? looks more like a txt file to me.
Another one, do we need to publish it if we publish the poetry pyproject.toml?

Reply:
`requirement.yaml` is removed in the final commit